src/vma/proto/route_val.h

```diff
@@ -37,12 +37,9 @@
 #include <netinet/in.h>
 #include <arpa/inet.h>
 
-#include "vma/dev/net_device_val.h"
-#include "vma/dev/net_device_table_mgr.h"
-
 #define BUFF_SIZE 255
 
-class route_val : public cache_observer
+class route_val
```

Comment:
cache_observer seem to be inherited previously for handling some event, however, currently route_val class is not handling any event.